rosidl_generator_cpp/cmake/rosidl_generator_cpp_generate_interfaces.cmake

```diff
@@ -90,6 +95,18 @@ add_custom_command(
   VERBATIM
 )
 
+# generate header to switch between export and import for a specific package
+set(_visibility_control_file
+  "${_output_path}/msg/rosidl_generator_cpp__visibility_control.hpp")
+string(TOUPPER "${PROJECT_NAME}" PROJECT_NAME_UPPER)
+configure_file(
+  "${rosidl_generator_cpp_TEMPLATE_DIR}/rosidl_generator_cpp__visibility_control.hpp.in"
+  "${_visibility_control_file}"
+  @ONLY
+)
+list(APPEND _generated_msg_headers "${_visibility_control_file}")
```

Comment:
@sloretz @clalancette I just noticed that this should probably be `_generated_headers` not `_generated_msg_headers`, it's not really critical because it's not required by any of the following targets I think but it still looks like a copy paste mistake to me. Sorry for that.

Reply:
Can you make a follow-up PR to fix it please?  Thanks.

Reply:
> Done in #755

Thank you, it is appreciated.